packages/mdc-checkbox/.npmignore

```diff
@@ -1,2 +1,3 @@
 *.ts
 !*.d.ts
+*_test.d.ts
```

Comment:
Also exclude all files in `test/` directory. I expect `test/` would have compiled JS files.